egui_glium/src/lib.rs

```diff
@@ -188,7 +188,7 @@ pub fn input_to_egui(
                 delta.x *= -1.0;
             }
 
-            if input_state.raw.modifiers.ctrl {
+            if input_state.raw.modifiers.command {
```

Comment:
```suggestion
            if input_state.raw.modifiers.ctrl || input_state.raw.modifiers.command {
```

So we also keep the old ctrl-scroll behavior. See https://github.com/emilk/egui/issues/401